src/Views/batchprocessor.html

```diff
@@ -15,7 +15,24 @@ <h4 class="modal-title">StreamStats Batch Processor</h4>
     <div style="padding-top: 10px;">
 
         <div id="submitBatch" ng-if="vm.selectedBatchProcessorTabName==='submitBatch'">
-            <p>This is the submit batch processor tab</p>
+            <div id="selectRegionID">
+                <label>Select State / Region:</label>
+
+                <select ng-model="selectedRegion" ng-options="x.name for x in vm.regionList | orderBy: 'name'"></select>
+            </div>
+            <div>                
+                <span>                
+                    <input id="cbDelineate" , type="checkbox" ng-checked="selectedRegion" onclick="return false;"

```

Comment:
When a state/region has been selected, we want the checkbox to be disabled. Rather than `onclick="return false;"`, let's use the `ng-disabled` directive: https://www.w3schools.com/angular/ng_ng-disabled.asp